src/components/Changelog/IncidentHistoryItemTypeContent.tsx

```diff
@@ -0,0 +1,121 @@
+import { FaComment } from "react-icons/fa";
+import {
+  IncidentHistory,
+  IncidentHistoryType
+} from "../../api/services/IncidentsHistory";
+import { hypothesisStatusIconMap } from "../../constants/hypothesisStatusOptions";
+import { Badge } from "../Badge";
+
+type IncidentHistoryItemTypeContentProps = {
+  incidentHistory: IncidentHistory;
+};
+
+const IncidentHistoryTypeToComponentMap = new Map<
+  `${IncidentHistoryType}`,
+  React.FC<IncidentHistoryItemTypeContentProps>
+>([
+  [
+    "incident.created",
+    ({ incidentHistory }) => {
+      return <>created Incident</>;
+    }
+  ],
+  [
+    "incident.status_updated",
+    ({ incidentHistory }) => {
+      return (
+        <>
+          marked as:{" "}
+          <Badge
+            className="inline-block"
+            text={incidentHistory.description?.toLocaleLowerCase()}
+          />
+        </>
+      );
+    }
+  ],
+  [
+    "responder.created",
+    ({ incidentHistory }) => {
+      const responder = incidentHistory.responder;
+      return <>Added responder: {responder?.person?.name} </>;
+    }
+  ],
+  [
+    "evidence.created",
+    ({ incidentHistory }) => {
+      const evidence = incidentHistory.evidence;
+      return (
+        <>
+          added <Badge className="inline-block" text={evidence?.description} />{" "}
+          evidence
+        </>
+      );
+    }
+  ],
+  [
+    "responder.commented",
+    ({ incidentHistory }) => {
+      const responder = incidentHistory.responder;
+      const comment = incidentHistory.comment;
+
+      return (
+        <>
+          <FaComment /> {comment?.comment} - {responder?.person?.name}{" "}
+        </>
+      );
+    }
+  ],
+  [
+    "hypothesis.created",
+    ({ incidentHistory }) => {
+      const hypothesis = incidentHistory.hypothesis;
+
+      const status =
+        incidentHistory.description as keyof typeof hypothesisStatusIconMap;
+      const icon = hypothesisStatusIconMap[status];
+
+      return (
+        <>
+          added <Badge className="inline-block" text={hypothesis?.title} />{" "}
```

Comment:
Shouldn't inline-block be the default className ? Can we also change the default class from px-2.5 to px-1